frontend/public/index.html

```diff
@@ -29,12 +29,6 @@
     <!-- initialize window.prerenderReady to false and then set to true in React app when app is ready for indexing -->
     <script> window.prerenderReady = false; </script>
 
-    <!-- load via script to enable web workers -->
-    <script
-      src="%PUBLIC_URL%/vendor/viz/viz-1.8.0.min.js"
-      type="javascript/worker"
-    ></script>
-
     <!-- Bokeh charts -->
```

Comment:
I wouldn't think we need this since we install `viz.js` as a transitive dependency of `d3-graphviz`, but let me know if I'm missing something